tasks/install.yml

```diff
@@ -0,0 +1,53 @@
+---
+
+# Configure nodejs
+- name: Install NodeJS apt key
+  apt_key:
+    url: https://deb.nodesource.com/gpgkey/nodesource.gpg.key
+    state: present
+
+- name: Add NodeJS apt repository
+  apt_repository:
+    repo: deb https://deb.nodesource.com/node_"{{ peertube_nodejs_version }}".x "{{ ansible_distribution_release }}" main
+    state: present
+    filename: nodesource
+
+- name: Install NodeJS
+  apt:
+    update_cache: yes
+    pkg: nodejs
+    state: present
+    
+- name: Install Yarn apt key
+  apt_key:
+    url: https://dl.yarnpkg.com/debian/pubkey.gpg
+    state: present
+
+- name: Add Yarn apt repo
+  apt_repository:
+    repo: deb https://dl.yarnpkg.com/debian/ stable main
+    state: present
+    filename: yarn
+
+- name: Install dependencies
+  apt:
+    name: '{{ item }}'
+    state: latest
+  with_items:
```

Comment:
Instead of using with-items, it is be better to use [loops](https://docs.ansible.com/ansible/latest/user_guide/playbooks_loops.html#with-items-)